Jenkinsfile

```diff
@@ -0,0 +1,18 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+node('nodejs6') {
+
+    stage('Checkout') {
+        checkout scm
+    }
+
+    stage('Lint') {
+        sh '''
+            npm install jsonlint
```

Comment:
Since this runs so frequently, we could consider baking jsonlint into Jenkins slave container image, to make this a tad quicker. It doesn't seem to take too long though.